src/Aspire.Dashboard/Components/Controls/ExceptionDetails.razor

```diff
@@ -29,6 +29,6 @@
 
     private void OnExceptionDetailsClicked(MouseEventArgs e)
```

Comment:
```suggestion
    private async Task OnExceptionDetailsClicked()
```
+ await opening the dialog.

I don't see a reason for this method not to be async.